### ISSUE TITLE
[sign_in_with_apple]FAILURE: Build failed with an exception.  

### ISSUE THREAD

**Issue by MichaelSowah** [aboutyou/dart_packages#96]
* What went wrong:                                                      
Execution failed for task ':app:checkDevDebugDuplicateClasses'.         
> 1 exception was raised by workers:                                    
  java.lang.RuntimeException: java.lang.RuntimeException: Duplicate class com.google.common.util.concurrent.ListenableFuture found in modules guava-20.0.jar (com.google.guava:guava:20.0) and listenablefuture-1.0.jar (com.google.guava:listenablefuture:1.0)

**Comment by HenriBeck** [comment]
@MichaelSowah the posted log doesn't show any indication that this would be caused by `sign_in_with_apple `.

On which version of `sign_in_with_apple ` are you and could you please post the full log of the build?

**Comment by sukhcha-in** [comment]
in App's Gradle file add:
```
defaultConfig {
        multiDexEnabled true
}
```
and 
```
dependencies {
    implementation 'com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava'
    implementation 'androidx.multidex:multidex:2.0.1'
}
```

**Comment by HenriBeck** [comment]
I'm closing this as it seems to have nothing to do with the `sign_in_with_apple` plugin

**Comment by bakua** [comment]
Btw the clash happens when you also use the https://pub.dev/packages/google_sign_in package. This is the one that includes com.google.guava:guava dependency.

I would assume there is a bunch of devs who run into this issue. It would be nice to either use guava directly as flutter team does or to mention this #96 in readme.

**Comment by HenriBeck** [comment]
@bakua Thanks for letting us know the exact reason and conflicting package. I can have a look next week that we can fix this hopefully in our manifest.

**Comment by HenriBeck** [comment]
Note that we are not using `guava` directly; we are using [androidx.browser](https://mvnrepository.com/artifact/androidx.browser/browser/1.2.0), which has a dependency on `guava`.

The only "fix" I was able to find is to use `1.0.0` of `androidx.browser` library which doesn't have this dependency.

**Comment by BerkSpar** [comment]
It doesn't work on flutter 2.0.5. I cannot sign-in with both Google and Apple. I tried to install ``androidx.browser`` on dependencies and it does not work.
